src/_mock/filters.js

```diff
@@ -0,0 +1,6 @@
+const filters = {
+  asset_types: (['Headsets']),
```

Comment:
Why the parenthesis? What about just having `asset_types: ['Headsets']`?

Reply:
Was trying to depict the fact that this is a `set` but it's still working fine as an array.